### PR TITLE
Create and setup of a new match

### PR DESCRIPTION
**Pull request by leandrost**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://service.feature-start-match.got-board-team-got-board-backend-d07a28673e235c051bbc.ttrcloud.com/' target='_blank'>Open stage</a> or <a href='http://service.feature-start-match.got-board-team-got-board-backend-d07a28673e235c051bbc.ttrcloud.com//teatro.log' target='_blank'>view logs</a>.

File: app/controllers/api/v1/matches_controller.rb [hunk 1]
```diff
@@ -1,6 +1,13 @@
 class Api::V1::MatchesController < ApplicationController
+
+  def index
+    #matches = Match.by_players(current_user.players)
```
Comment (leandrost) [inline app/controllers/api/v1/matches_controller.rb]: Codacy Issue found: [Missing space after #.](https://www.codacy.com/app/leandrosthimoteo_2996/got-board-api/file/2347570957/issues/source?bid=3216569&fileBranchId=3216571#l4)

File: app/controllers/api/v1/players_controller.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class Api::V1::PlayersController < ApplicationController
+  def update
+    player = Player.find(params[:id])
```
Comment (leandrost) [inline app/controllers/api/v1/players_controller.rb]: Codacy Issue found: [Unscoped call to Player#find](https://www.codacy.com/app/leandrosthimoteo_2996/got-board-api/file/2347570959/issues/source?bid=3216569&fileBranchId=3216571#l3)

File: app/models/match.rb [hunk 3]
```diff
@@ -9,4 +9,9 @@ class Match < ActiveRecord::Base
   # has_one :westeros_three_deck
   # has_one :tides_of_battle_deck # optional
 
+  scope :by_players, lambda { |player_ids| includes(:players).where(players: { id: player_ids }) }
```
Comment (leandrost) [inline app/models/match.rb]: Codacy Issue found: [Use the new lambda literal syntax `->(params) {...}`.](https://www.codacy.com/app/leandrosthimoteo_2996/got-board-api/file/2347570994/issues/source?bid=3216569&fileBranchId=3216571#l12)

File: app/controllers/api/v1/players_controller.rb (hunk 2 above)
Comment (leandrost) [inline app/controllers/api/v1/players_controller.rb]: Codacy Issue found: [Unscoped call to Player#find](https://www.codacy.com/app/leandrosthimoteo_2996/got-board-api/file/2347584068/issues/source?bid=3216569&fileBranchId=3216571#l3)

File: app/models/match.rb (hunk 3 above)
Comment (leandrost) [inline app/models/match.rb]: Codacy Issue found: [Use the new lambda literal syntax `->(params) {...}`.](https://www.codacy.com/app/leandrosthimoteo_2996/got-board-api/file/2347584103/issues/source?bid=3216569&fileBranchId=3216571#l12)

File: app/controllers/api/v1/matches_controller.rb (hunk 1 above)
Comment (leandrost) [inline app/controllers/api/v1/matches_controller.rb]: Codacy Issue found: [Missing space after #.](https://www.codacy.com/app/leandrosthimoteo_2996/got-board-api/file/2347584066/issues/source?bid=3216569&fileBranchId=3216571#l4)

Comment (leandrost) [inline app/controllers/api/v1/matches_controller.rb]: Codacy Issue found: [Missing space after #.](https://www.codacy.com/app/leandrosthimoteo_2996/got-board-api/file/2347594642/issues/source?bid=3216569&fileBranchId=3216571#l4)

File: app/models/match.rb (hunk 3 above)
Comment (leandrost) [inline app/models/match.rb]: Codacy Issue found: [Use the new lambda literal syntax `->(params) {...}`.](https://www.codacy.com/app/leandrosthimoteo_2996/got-board-api/file/2347594679/issues/source?bid=3216569&fileBranchId=3216571#l12)

File: app/controllers/api/v1/players_controller.rb (hunk 2 above)
Comment (leandrost) [inline app/controllers/api/v1/players_controller.rb]: Codacy Issue found: [Unscoped call to Player#find](https://www.codacy.com/app/leandrosthimoteo_2996/got-board-api/file/2347594644/issues/source?bid=3216569&fileBranchId=3216571#l3)

Comment (leandrost) [inline app/controllers/api/v1/players_controller.rb]: Codacy Issue found: [Unscoped call to Player#find](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2362082069/issues/source?bid=3216569&fileBranchId=3216571#l3)

File: app/models/match.rb (hunk 3 above)
Comment (leandrost) [inline app/models/match.rb]: Codacy Issue found: [Use the new lambda literal syntax `->(params) {...}`.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2362082104/issues/source?bid=3216569&fileBranchId=3216571#l12)

File: app/controllers/api/v1/matches_controller.rb (hunk 1 above)
Comment (leandrost) [inline app/controllers/api/v1/matches_controller.rb]: Codacy Issue found: [Missing space after #.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2362082067/issues/source?bid=3216569&fileBranchId=3216571#l4)

File: app/controllers/api/v1/players_controller.rb (hunk 2 above)
Comment (leandrost) [inline app/controllers/api/v1/players_controller.rb]: Codacy Issue found: [Unscoped call to Player#find](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2382003785/issues/source?bid=3216569&fileBranchId=3216571#l3)

File: app/models/match.rb (hunk 3 above)
Comment (leandrost) [inline app/models/match.rb]: Codacy Issue found: [Use the new lambda literal syntax `->(params) {...}`.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2382003820/issues/source?bid=3216569&fileBranchId=3216571#l12)

File: app/controllers/api/v1/matches_controller.rb (hunk 1 above)
Comment (leandrost) [inline app/controllers/api/v1/matches_controller.rb]: Codacy Issue found: [Missing space after #.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2382003783/issues/source?bid=3216569&fileBranchId=3216571#l4)

File: app/models/match.rb (hunk 3 above)
Comment (leandrost) [inline app/models/match.rb]: Codacy Issue found: [Use the new lambda literal syntax `->(params) {...}`.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2382753723/issues/source?bid=3216569&fileBranchId=3216571#l12)

File: app/controllers/api/v1/matches_controller.rb (hunk 1 above)
Comment (leandrost) [inline app/controllers/api/v1/matches_controller.rb]: Codacy Issue found: [Missing space after #.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2382753686/issues/source?bid=3216569&fileBranchId=3216571#l4)

File: app/controllers/api/v1/players_controller.rb (hunk 2 above)
Comment (leandrost) [inline app/controllers/api/v1/players_controller.rb]: Codacy Issue found: [Unscoped call to Player#find](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2382753688/issues/source?bid=3216569&fileBranchId=3216571#l3)

File: app/interactions/matches/create_match.rb [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+    HouseCard.create!(name: "Greatjon Umber", player: player)
+    HouseCard.create!(name: "The Blackfish", player: player)
+    HouseCard.create!(name: "Ser Rodrick Cassel", player: player)
+    HouseCard.create!(name: "Catelyn Stark", player: player)
+  end
+
+  def create_greyjoy_cards(player)
+  end
+
+  def create_tyrell_cards(player)
```
Comment (leandrost) [inline app/interactions/matches/create_match.rb]: Codacy Issue found: [Unused method argument - `player`. If it's necessary, use `_` or `_player` as an argument name to indicate that it won't be used. You can also write as `create_tyrell_cards(*)` if you want the method to accept any arguments but don't care about them.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2382870506/issues/source?bid=3216569&fileBranchId=3216571#l172)

File: app/interactions/matches/create_match.rb [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+    less_than_or_equal_to: 6
+
+  def execute
+    ActiveRecord::Base.transaction do
+      match = Match.create!
+      Board.create!(match: match)
+      create_players(match, players_count)
+      create_power_pool(match)
+      distribute_player_things(match)
+      #create_decks(match, use_tides_of_battle_cards)
```
Comment (leandrost) [inline app/interactions/matches/create_match.rb]: Codacy Issue found: [Missing space after #.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2382870506/issues/source?bid=3216569&fileBranchId=3216571#l15)

File: app/interactions/matches/create_match.rb [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+  def create_decks(match, use_tides_of_battle_cards)
+    # TODO
+    # create_westeros_deck_one(match)
+    # create_westeros_deck_two(match)
+    # create_westeros_deck_three(match)
+    # create_wildlings_deck(match)
+    # create_tides_of_battle_deck(match) if use_tides_of_battle_cards
+  end
+
+  def create_power_pool(match)
```
Comment (leandrost) [inline app/interactions/matches/create_match.rb]: Codacy Issue found: [Unused method argument - `match`. If it's necessary, use `_` or `_match` as an argument name to indicate that it won't be used. You can also write as `create_power_pool(*)` if you want the method to accept any arguments but don't care about them.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2382870506/issues/source?bid=3216569&fileBranchId=3216571#l118)

File: app/interactions/matches/create_match.rb [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+      # RoundTrack,                # TODO
+      # KingsCourtInfluenceTrack,  # TODO
+      # IronThroneInfluenceTrack,  # TODO
+      # FiefdomsInfluenceTrack     # TODO
+    ].each do |track|
+      track.setup!(match, board, players)
+    end
+  end
+
+  def create_decks(match, use_tides_of_battle_cards)
```
Comment (leandrost) [inline app/interactions/matches/create_match.rb]: Codacy Issue found: [Unused method argument - `match`. If it's necessary, use `_` or `_match` as an argument name to indicate that it won't be used. You can also write as `create_decks(*)` if you want the method to accept any arguments but don't care about them.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2382870506/issues/source?bid=3216569&fileBranchId=3216571#l109)

File: app/interactions/matches/create_match.rb [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+    HouseCard.create!(name: "Catelyn Stark", player: player)
+  end
+
+  def create_greyjoy_cards(player)
+  end
+
+  def create_tyrell_cards(player)
+  end
+
+  def create_martell_cards(player)
```
Comment (leandrost) [inline app/interactions/matches/create_match.rb]: Codacy Issue found: [Unused method argument - `player`. If it's necessary, use `_` or `_player` as an argument name to indicate that it won't be used. You can also write as `create_martell_cards(*)` if you want the method to accept any arguments but don't care about them.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2382870506/issues/source?bid=3216569&fileBranchId=3216571#l175)

File: app/interactions/matches/create_match.rb [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+    SupportP.create!(player: player, board: board)
+  end
+
+  def create_power_tokens(player)
+    5.times do
+      PowerToken.create!(player: player)
+    end
+  end
+
+  def create_baratheon_cards(player)
```
Comment (leandrost) [inline app/interactions/matches/create_match.rb]: Codacy Issue found: [Unused method argument - `player`. If it's necessary, use `_` or `_player` as an argument name to indicate that it won't be used. You can also write as `create_baratheon_cards(*)` if you want the method to accept any arguments but don't care about them.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2382870506/issues/source?bid=3216569&fileBranchId=3216571#l153)

File: app/interactions/matches/create_match.rb [hunk 10]
```diff
@@ -0,0 +1,178 @@
...
+  end
+
+  def create_lannister_cards(player)
```
Comment (leandrost) [inline app/interactions/matches/create_match.rb]: Codacy Issue found: [Unused method argument - `player`. If it's necessary, use `_` or `_player` as an argument name to indicate that it won't be used. You can also write as `create_lannister_cards(*)` if you want the method to accept any arguments but don't care about them.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2382870506/issues/source?bid=3216569&fileBranchId=3216571#l156)

File: app/interactions/matches/create_match.rb [hunk 11]
```diff
@@ -0,0 +1,178 @@
...
+    HouseCard.create!(name: "Eddard Stark", player: player)
+    HouseCard.create!(name: "Robb Stark", player: player)
+    HouseCard.create!(name: "Roose Bolton", player: player)
+    HouseCard.create!(name: "Greatjon Umber", player: player)
+    HouseCard.create!(name: "The Blackfish", player: player)
+    HouseCard.create!(name: "Ser Rodrick Cassel", player: player)
+    HouseCard.create!(name: "Catelyn Stark", player: player)
+  end
+
+  def create_greyjoy_cards(player)
```
Comment (leandrost) [inline app/interactions/matches/create_match.rb]: Codacy Issue found: [Unused method argument - `player`. If it's necessary, use `_` or `_player` as an argument name to indicate that it won't be used. You can also write as `create_greyjoy_cards(*)` if you want the method to accept any arguments but don't care about them.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2382870506/issues/source?bid=3216569&fileBranchId=3216571#l169)

File: app/models/match.rb (hunk 3 above)
Comment (leandrost) [inline app/models/match.rb]: Codacy Issue found: [Use the new lambda literal syntax `->(params) {...}`.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2382870536/issues/source?bid=3216569&fileBranchId=3216571#l12)

File: app/controllers/api/v1/players_controller.rb (hunk 2 above)
Comment (leandrost) [inline app/controllers/api/v1/players_controller.rb]: Codacy Issue found: [Unscoped call to Player#find](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2382870500/issues/source?bid=3216569&fileBranchId=3216571#l3)

File: app/interactions/matches/create_match.rb [hunk 12]
```diff
@@ -0,0 +1,115 @@
...
+  def create_power_tokens(player)
+    5.times do
+      PowerToken.create!(player: player)
+    end
+  end
+
+  def create_baratheon_cards(player)
+  end
+
+  def create_lannister_cards(player)
```
Comment (leandrost) [inline app/interactions/matches/create_match.rb]: Codacy Issue found: [Unused method argument - `player`. If it's necessary, use `_` or `_player` as an argument name to indicate that it won't be used. You can also write as `create_lannister_cards(*)` if you want the method to accept any arguments but don't care about them.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2383240825/issues/source?bid=3216569&fileBranchId=3216571#l94)

File: app/interactions/matches/create_match.rb [hunk 13]
```diff
@@ -0,0 +1,115 @@
...
+    HouseCard.create!(name: "Catelyn Stark", player: player)
+  end
+
+  def create_greyjoy_cards(player)
+  end
+
+  def create_tyrell_cards(player)
+  end
+
+  def create_martell_cards(player)
```
Comment (leandrost) [inline app/interactions/matches/create_match.rb]: Codacy Issue found: [Unused method argument - `player`. If it's necessary, use `_` or `_player` as an argument name to indicate that it won't be used. You can also write as `create_martell_cards(*)` if you want the method to accept any arguments but don't care about them.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2383240825/issues/source?bid=3216569&fileBranchId=3216571#l113)

File: app/interactions/matches/create_match.rb [hunk 14]
```diff
@@ -0,0 +1,115 @@
...
+    HouseCard.create!(name: "Eddard Stark", player: player)
+    HouseCard.create!(name: "Robb Stark", player: player)
+    HouseCard.create!(name: "Roose Bolton", player: player)
+    HouseCard.create!(name: "Greatjon Umber", player: player)
+    HouseCard.create!(name: "The Blackfish", player: player)
+    HouseCard.create!(name: "Ser Rodrick Cassel", player: player)
+    HouseCard.create!(name: "Catelyn Stark", player: player)
+  end
+
+  def create_greyjoy_cards(player)
```
Comment (leandrost) [inline app/interactions/matches/create_match.rb]: Codacy Issue found: [Unused method argument - `player`. If it's necessary, use `_` or `_player` as an argument name to indicate that it won't be used. You can also write as `create_greyjoy_cards(*)` if you want the method to accept any arguments but don't care about them.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2383240825/issues/source?bid=3216569&fileBranchId=3216571#l107)

File: app/interactions/matches/create_match.rb [hunk 15]
```diff
@@ -0,0 +1,115 @@
...
+  end
+
+  def create_tyrell_cards(player)
```
Comment (leandrost) [inline app/interactions/matches/create_match.rb]: Codacy Issue found: [Unused method argument - `player`. If it's necessary, use `_` or `_player` as an argument name to indicate that it won't be used. You can also write as `create_tyrell_cards(*)` if you want the method to accept any arguments but don't care about them.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2383240825/issues/source?bid=3216569&fileBranchId=3216571#l110)

File: app/interactions/matches/create_match.rb [hunk 16]
```diff
@@ -0,0 +1,115 @@
...
+  def create_decks(match, use_tides_of_battle_cards)
+    # TODO
+    # create_westeros_deck_one(match)
+    # create_westeros_deck_two(match)
+    # create_westeros_deck_three(match)
+    # create_wildlings_deck(match)
+    # create_tides_of_battle_deck(match) if use_tides_of_battle_cards
+  end
+
+  def create_power_pool(match)
```
Comment (leandrost) [inline app/interactions/matches/create_match.rb]: Codacy Issue found: [Unused method argument - `match`. If it's necessary, use `_` or `_match` as an argument name to indicate that it won't be used. You can also write as `create_power_pool(*)` if you want the method to accept any arguments but don't care about them.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2383240825/issues/source?bid=3216569&fileBranchId=3216571#l64)

File: app/interactions/matches/create_match.rb [hunk 17]
```diff
@@ -0,0 +1,115 @@
...
+      # RoundTrack,                # TODO
+      # KingsCourtInfluenceTrack,  # TODO
+      # IronThroneInfluenceTrack,  # TODO
+      # FiefdomsInfluenceTrack     # TODO
+    ].each do |track|
+      track.setup!(match, board, players)
+    end
+  end
+
+  def create_decks(match, use_tides_of_battle_cards)
```
Comment (leandrost) [inline app/interactions/matches/create_match.rb]: Codacy Issue found: [Unused method argument - `match`. If it's necessary, use `_` or `_match` as an argument name to indicate that it won't be used. You can also write as `create_decks(*)` if you want the method to accept any arguments but don't care about them.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2383240825/issues/source?bid=3216569&fileBranchId=3216571#l55)

File: app/interactions/matches/create_match.rb [hunk 18]
```diff
@@ -0,0 +1,115 @@
...
+    SupportP.create!(player: player, board: board)
+  end
+
+  def create_power_tokens(player)
+    5.times do
+      PowerToken.create!(player: player)
+    end
+  end
+
+  def create_baratheon_cards(player)
```
Comment (leandrost) [inline app/interactions/matches/create_match.rb]: Codacy Issue found: [Unused method argument - `player`. If it's necessary, use `_` or `_player` as an argument name to indicate that it won't be used. You can also write as `create_baratheon_cards(*)` if you want the method to accept any arguments but don't care about them.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2383240825/issues/source?bid=3216569&fileBranchId=3216571#l91)

File: app/controllers/api/v1/players_controller.rb (hunk 2 above)
Comment (leandrost) [inline app/controllers/api/v1/players_controller.rb]: Codacy Issue found: [Unscoped call to Player#find](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2383240819/issues/source?bid=3216569&fileBranchId=3216571#l3)

File: app/models/match.rb (hunk 3 above)
Comment (leandrost) [inline app/models/match.rb]: Codacy Issue found: [Use the new lambda literal syntax `->(params) {...}`.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2383240856/issues/source?bid=3216569&fileBranchId=3216571#l12)

File: app/interactions/matches/create_match.rb [hunk 19]
```diff
@@ -0,0 +1,88 @@
...
+  def create_decks(match, use_tides_of_battle_cards)
+    # TODO
+    # create_westeros_deck_one(match)
+    # create_westeros_deck_two(match)
+    # create_westeros_deck_three(match)
+    # create_wildlings_deck(match)
+    # create_tides_of_battle_deck(match) if use_tides_of_battle_cards
+  end
+
+  def create_power_pool(match)
```
Comment (leandrost) [inline app/interactions/matches/create_match.rb]: Codacy Issue found: [Unused method argument - `match`. If it's necessary, use `_` or `_match` as an argument name to indicate that it won't be used. You can also write as `create_power_pool(*)` if you want the method to accept any arguments but don't care about them.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2384555189/issues/source?bid=3216569&fileBranchId=3216571#l64)

File: app/interactions/matches/create_match.rb [hunk 20]
```diff
@@ -0,0 +1,88 @@
...
+      # RoundTrack,                # TODO
+      # KingsCourtInfluenceTrack,  # TODO
+      # IronThroneInfluenceTrack,  # TODO
+      # FiefdomsInfluenceTrack     # TODO
+    ].each do |track|
+      track.setup!(match, board, players)
+    end
+  end
+
+  def create_decks(match, use_tides_of_battle_cards)
```
Comment (leandrost) [inline app/interactions/matches/create_match.rb]: Codacy Issue found: [Unused method argument - `match`. If it's necessary, use `_` or `_match` as an argument name to indicate that it won't be used. You can also write as `create_decks(*)` if you want the method to accept any arguments but don't care about them.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2384555189/issues/source?bid=3216569&fileBranchId=3216571#l55)

File: app/controllers/api/v1/players_controller.rb (hunk 2 above)
Comment (leandrost) [inline app/controllers/api/v1/players_controller.rb]: Codacy Issue found: [Unscoped call to Player#find](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2384555183/issues/source?bid=3216569&fileBranchId=3216571#l3)

File: app/models/match.rb (hunk 3 above)
Comment (leandrost) [inline app/models/match.rb]: Codacy Issue found: [Use the new lambda literal syntax `->(params) {...}`.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2384555221/issues/source?bid=3216569&fileBranchId=3216571#l12)

File: app/interactions/matches/create_match.rb [hunk 21]
```diff
@@ -0,0 +1,70 @@
...
+  # (Rulebook, page 28)
+  def create_players(match, number_of_players)
+    house_names = Player::HOUSES.first(number_of_players)
+    house_names.each do |house_name|
+      player = match.players.create!(match: match, house: house_name)
+      distribute_player_things(player, match.board)
+    end
+  end
+
+  def distribute_player_things(player, board)
```
Comment (leandrost) [inline app/interactions/matches/create_match.rb]: Codacy Issue found: [Unused method argument - `board`. If it's necessary, use `_` or `_board` as an argument name to indicate that it won't be used.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2451703218/issues/source?bid=3216569&fileBranchId=3216571#l34)

File: app/interactions/matches/create_match.rb [hunk 22]
```diff
@@ -0,0 +1,70 @@
...
+  def create_decks(match, use_tides_of_battle_cards)
+    # TODO
+    # create_westeros_deck_one(match)
+    # create_westeros_deck_two(match)
+    # create_westeros_deck_three(match)
+    # create_wildlings_deck(match)
+    # create_tides_of_battle_deck(match) if use_tides_of_battle_cards
+  end
+
+  def create_power_pool(match)
```
Comment (leandrost) [inline app/interactions/matches/create_match.rb]: Codacy Issue found: [Unused method argument - `match`. If it's necessary, use `_` or `_match` as an argument name to indicate that it won't be used. You can also write as `create_power_pool(*)` if you want the method to accept any arguments but don't care about them.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2451703218/issues/source?bid=3216569&fileBranchId=3216571#l64)

File: app/interactions/matches/create_match.rb [hunk 23]
```diff
@@ -0,0 +1,70 @@
...
+      # RoundTrack,                # TODO
+      # KingsCourtInfluenceTrack,  # TODO
+      # IronThroneInfluenceTrack,  # TODO
+      # FiefdomsInfluenceTrack     # TODO
+    ].each do |track|
+      track.setup!(match, board, players)
+    end
+  end
+
+  def create_decks(match, use_tides_of_battle_cards)
```
Comment (leandrost) [inline app/interactions/matches/create_match.rb]: Codacy Issue found: [Unused method argument - `match`. If it's necessary, use `_` or `_match` as an argument name to indicate that it won't be used. You can also write as `create_decks(*)` if you want the method to accept any arguments but don't care about them.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2451703218/issues/source?bid=3216569&fileBranchId=3216571#l55)

File: app/controllers/api/v1/players_controller.rb (hunk 2 above)
Comment (leandrost) [inline app/controllers/api/v1/players_controller.rb]: Codacy Issue found: [Unscoped call to Player#find](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2451703208/issues/source?bid=3216569&fileBranchId=3216571#l3)

File: app/interactions/matches/create_match.rb (hunk 21 above)
Comment (leandrost) [inline app/interactions/matches/create_match.rb:34]: Codacy Issue found: [Unused method argument - `board`. If it's necessary, use `_` or `_board` as an argument name to indicate that it won't be used.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2460569684/issues/source?bid=3216569&fileBranchId=3216571#l34)

File: app/interactions/matches/create_match.rb (hunk 22 above)
Comment (leandrost) [inline app/interactions/matches/create_match.rb:64]: Codacy Issue found: [Unused method argument - `match`. If it's necessary, use `_` or `_match` as an argument name to indicate that it won't be used. You can also write as `create_power_pool(*)` if you want the method to accept any arguments but don't care about them.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2460569684/issues/source?bid=3216569&fileBranchId=3216571#l64)

File: app/interactions/matches/create_match.rb (hunk 23 above)
Comment (leandrost) [inline app/interactions/matches/create_match.rb:55]: Codacy Issue found: [Unused method argument - `match`. If it's necessary, use `_` or `_match` as an argument name to indicate that it won't be used. You can also write as `create_decks(*)` if you want the method to accept any arguments but don't care about them.](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2460569684/issues/source?bid=3216569&fileBranchId=3216571#l55)

File: app/controllers/api/v1/players_controller.rb (hunk 2 above)
Comment (leandrost) [inline app/controllers/api/v1/players_controller.rb:3]: Codacy Issue found: [Unscoped call to Player#find](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2460569678/issues/source?bid=3216569&fileBranchId=3216571#l3)

**Pull request merged**